log/logstream/logstream_test.go

```diff
@@ -0,0 +1,68 @@
+package logstream
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Let's not add dependencies everywhere.. This package is small enough to be tested without testify.